manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -355,6 +357,8 @@ spec:
       workspace: build-workspace-pv
   workspaces:
   - name: build-workspace-pv
+  - name: git-secret
+    optional: true
```

Comment:
I am not 100% sure I understand what you mean. 

Are you saying that from a users perspective you can add you git credentials like so (or something close):
```yaml
"https://{username}:{github_pat_token}@github.com
https://{username}:{gitlab_pat_token}@gitlab.com
"
```

But for the e2e tests we just use one set of a username/token because we are enforcing they are from the same domain?

Reply:
@jackdelahunt yes, exactly.
1. We can have just one secret for many repositories.
2. We will anyway use just one par of user/token credentials to simplify our test setup, i.e. it is not needed to be able to set separate user/token for Containerfile Git repo and for Model Git repo in our use case. That will simplify the test setup, i.e. no need for building a more complex string with more than one username/token URL etc.